autoload/lsp/ui/vim/signature_help.vim

```diff
@@ -110,21 +111,30 @@ function! s:get_parameter_doc(parameter) abort
     return printf('***%s*** - %s', a:parameter['label'], l:doc)
 endfunction
 
-function! s:insert_char_pre() abort
-    let l:buf = bufnr('%')
-    for l:server_name in lsp#get_whitelisted_servers(l:buf)
-        let l:keys = lsp#capabilities#get_signature_help_trigger_characters(l:server_name)
-        for l:key in l:keys
-            if l:key ==# v:char
-                call timer_start(0, {_-> lsp#ui#vim#signature_help#get_signature_help_under_cursor() })
-            endif
-        endfor
+function! s:on_text_changed() abort
+    let l:bufnr = bufnr('%')
+    call timer_stop(s:debounce_timer_id)
+    let s:debounce_timer_id = timer_start(500, { -> s:on_text_changed_after(l:bufnr) })
```

Comment:
any reason why it is 500? Can we lower this to 250?

Reply:
Yes. We can use less than 500 value (and I changed to this value to 200).
This is not meaningful value for user, so I didn't put the option to it.